docs/gettingStarted/ALL.md

```diff
@@ -81,6 +83,8 @@ Next up, we'll populate the `zeplinNames` key.
 
 ### Connect to component from Zeplin
 
+> **`zeplinNames` is deprecated. It is recommended to use [Zeplin CLI INITIALIZE commands or VS Code Extension](./INITIALIZE.md) to add `zeplinIds` into the configuration file.**
```

Comment:
Is `zeplinNames` deprecated or just a way that is not suggested if they are using `identifier` based update for components?